{"@t":"2022-09-12T12:55:33.2094234Z","@mt":"Starting web host"}
{"@t":"2022-09-12T12:55:34.4097244Z","@mt":"Host terminated unexpectedly","@l":"Fatal","@x":"Microsoft.Extensions.Hosting.HostFactoryResolver+HostingListener+StopTheHostException: Exception of type 'Microsoft.Extensions.Hosting.HostFactoryResolver+HostingListener+StopTheHostException' was thrown.\r\n   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)\r\n   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)\r\n   at Microsoft.Extensions.Hosting.HostBuilder.Build()\r\n   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()\r\n   at TweetApp.Backend.Program.Main(String[] args) in C:\\Users\\cogdotnet1008\\Downloads\\Component_2\\Backend\\TweetApp.Backend\\Program.cs:line 108"}
{"@t":"2022-09-12T12:56:26.4373254Z","@mt":"Starting web host"}
{"@t":"2022-09-12T12:56:29.8374318Z","@mt":"Now listening on: {address}","address":"https://localhost:7149","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T12:56:29.8493921Z","@mt":"Now listening on: {address}","address":"http://localhost:5149","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T12:56:29.8571539Z","@mt":"Application started. Press Ctrl+C to shut down.","SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T12:56:29.8595258Z","@mt":"Hosting environment: {envName}","envName":"Development","SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T12:56:29.8620450Z","@mt":"Content root path: {contentRoot}","contentRoot":"C:\\Users\\cogdotnet1008\\Downloads\\Component_2\\Backend\\TweetApp.Backend\\","SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T12:56:44.1571517Z","@mt":"Entity Framework Core {version} initialized '{contextType}' using provider '{provider}:{providerVersion}' with options: {options}","version":"6.0.8","contextType":"ApplicationDbContext","provider":"Microsoft.EntityFrameworkCore.SqlServer","providerVersion":"6.0.8","options":"None","EventId":{"Id":10403,"Name":"Microsoft.EntityFrameworkCore.Infrastructure.ContextInitialized"},"SourceContext":"Microsoft.EntityFrameworkCore.Infrastructure","ActionId":"43c1efdc-96e4-4c12-b0ea-194a9dbcd32e","ActionName":"TweetApp.Backend.Controllers.UsersController.Register (TweetApp.Backend)","RequestId":"0HMKKNOR2D8IP:00000001","RequestPath":"/api/v1/tweets/register","ConnectionId":"0HMKKNOR2D8IP"}
{"@t":"2022-09-12T12:56:50.9542930Z","@mt":"HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms","@r":["8953.8353"],"RequestMethod":"POST","RequestPath":"/api/v1/tweets/register","StatusCode":200,"Elapsed":8953.8353,"SourceContext":"Serilog.AspNetCore.RequestLoggingMiddleware","ActionId":"43c1efdc-96e4-4c12-b0ea-194a9dbcd32e","ActionName":"TweetApp.Backend.Controllers.UsersController.Register (TweetApp.Backend)","RequestId":"0HMKKNOR2D8IP:00000001","ConnectionId":"0HMKKNOR2D8IP"}
{"@t":"2022-09-12T12:57:10.8825568Z","@mt":"Entity Framework Core {version} initialized '{contextType}' using provider '{provider}:{providerVersion}' with options: {options}","version":"6.0.8","contextType":"ApplicationDbContext","provider":"Microsoft.EntityFrameworkCore.SqlServer","providerVersion":"6.0.8","options":"None","EventId":{"Id":10403,"Name":"Microsoft.EntityFrameworkCore.Infrastructure.ContextInitialized"},"SourceContext":"Microsoft.EntityFrameworkCore.Infrastructure","ActionId":"277590ff-43b9-49bb-b253-d02d92aff11b","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKNOR2D8IQ:00000001","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKKNOR2D8IQ"}
{"@t":"2022-09-12T12:57:11.5716562Z","@mt":"HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms","@r":["751.3332"],"RequestMethod":"POST","RequestPath":"/api/v1/tweets/login","StatusCode":200,"Elapsed":751.3332,"SourceContext":"Serilog.AspNetCore.RequestLoggingMiddleware","ActionId":"277590ff-43b9-49bb-b253-d02d92aff11b","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKNOR2D8IQ:00000001","ConnectionId":"0HMKKNOR2D8IQ"}
{"@t":"2022-09-12T12:58:03.6342261Z","@mt":"Starting web host"}
{"@t":"2022-09-12T12:58:49.0305463Z","@mt":"Starting web host"}
{"@t":"2022-09-12T12:58:52.8462857Z","@mt":"Storing keys in a directory '{path}' that may not be persisted outside of the container. Protected data will be unavailable when container is destroyed.","@l":"Warning","path":"C:\\Users\\ContainerUser\\AppData\\Local\\ASP.NET\\DataProtection-Keys","EventId":{"Id":60,"Name":"UsingEphemeralFileSystemLocationInContainer"},"SourceContext":"Microsoft.AspNetCore.DataProtection.Repositories.FileSystemXmlRepository"}
{"@t":"2022-09-12T12:58:53.6592560Z","@mt":"Now listening on: {address}","address":"https://[::]:443","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T12:58:53.6597557Z","@mt":"Now listening on: {address}","address":"http://[::]:80","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T12:58:53.6843646Z","@mt":"Application started. Press Ctrl+C to shut down.","SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T12:58:53.6850787Z","@mt":"Hosting environment: {envName}","envName":"Development","SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T12:58:53.6854846Z","@mt":"Content root path: {contentRoot}","contentRoot":"C:\\app\\","SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T12:59:07.8713217Z","@mt":"Entity Framework Core {version} initialized '{contextType}' using provider '{provider}:{providerVersion}' with options: {options}","version":"6.0.8","contextType":"ApplicationDbContext","provider":"Microsoft.EntityFrameworkCore.SqlServer","providerVersion":"6.0.8","options":"None","EventId":{"Id":10403,"Name":"Microsoft.EntityFrameworkCore.Infrastructure.ContextInitialized"},"SourceContext":"Microsoft.EntityFrameworkCore.Infrastructure","ActionId":"4ba95a3c-2d2e-4fa7-b686-aa46ac338dae","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKNQ6I7E7G:00000001","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKKNQ6I7E7G"}
{"@t":"2022-09-12T12:59:10.4240068Z","@mt":"An error occurred using the connection to database '{database}' on server '{server}'.","@l":"Error","database":"tweetappapi","server":"tcp:tweetapp-server-atman.database.windows.net,1433","EventId":{"Id":20004,"Name":"Microsoft.EntityFrameworkCore.Database.Connection.ConnectionError"},"SourceContext":"Microsoft.EntityFrameworkCore.Database.Connection","ActionId":"4ba95a3c-2d2e-4fa7-b686-aa46ac338dae","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKNQ6I7E7G:00000001","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKKNQ6I7E7G"}
{"@t":"2022-09-12T12:59:10.7029934Z","@mt":"An exception occurred while iterating over the results of a query for context type '{contextType}'.{newline}{error}","@l":"Error","@x":"System.InvalidOperationException: An exception has been raised that is likely due to a transient failure. Consider enabling transient error resiliency by adding 'EnableRetryOnFailure' to the 'UseSqlServer' call.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): The instance of SQL Server you attempted to connect to requires encryption but this machine does not support it.\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CheckPoolBlockingPeriod(Exception e)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(AsyncEnumerator enumerator, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\nClientConnectionId:aff964af-6101-41f7-829b-7c794ed2de1a\r\nError Number:20,State:0,Class:20\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()","contextType":"TweetApp.Backend.Repository.ApplicationDbContext","newline":"\r\n","error":"System.InvalidOperationException: An exception has been raised that is likely due to a transient failure. Consider enabling transient error resiliency by adding 'EnableRetryOnFailure' to the 'UseSqlServer' call.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): The instance of SQL Server you attempted to connect to requires encryption but this machine does not support it.\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CheckPoolBlockingPeriod(Exception e)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(AsyncEnumerator enumerator, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\nClientConnectionId:aff964af-6101-41f7-829b-7c794ed2de1a\r\nError Number:20,State:0,Class:20\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()","EventId":{"Id":10100,"Name":"Microsoft.EntityFrameworkCore.Query.QueryIterationFailed"},"SourceContext":"Microsoft.EntityFrameworkCore.Query","ActionId":"4ba95a3c-2d2e-4fa7-b686-aa46ac338dae","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKNQ6I7E7G:00000001","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKKNQ6I7E7G"}
{"@t":"2022-09-12T12:59:10.9808685Z","@mt":"Invalid username or password","@l":"Error","SourceContext":"TweetApp.Backend.Controllers.UsersController","ActionId":"4ba95a3c-2d2e-4fa7-b686-aa46ac338dae","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKNQ6I7E7G:00000001","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKKNQ6I7E7G"}
{"@t":"2022-09-12T12:59:12.7447621Z","@mt":"HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms","@r":["7216.1783"],"RequestMethod":"POST","RequestPath":"/api/v1/tweets/login","StatusCode":200,"Elapsed":7216.1783,"SourceContext":"Serilog.AspNetCore.RequestLoggingMiddleware","ActionId":"4ba95a3c-2d2e-4fa7-b686-aa46ac338dae","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKNQ6I7E7G:00000001","ConnectionId":"0HMKKNQ6I7E7G"}
{"@t":"2022-09-12T13:15:29.4735282Z","@mt":"Entity Framework Core {version} initialized '{contextType}' using provider '{provider}:{providerVersion}' with options: {options}","version":"6.0.8","contextType":"ApplicationDbContext","provider":"Microsoft.EntityFrameworkCore.SqlServer","providerVersion":"6.0.8","options":"None","EventId":{"Id":10403,"Name":"Microsoft.EntityFrameworkCore.Infrastructure.ContextInitialized"},"SourceContext":"Microsoft.EntityFrameworkCore.Infrastructure","ActionId":"4ba95a3c-2d2e-4fa7-b686-aa46ac338dae","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKNQ6I7E7I:00000001","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKKNQ6I7E7I"}
{"@t":"2022-09-12T13:15:30.1793565Z","@mt":"An error occurred using the connection to database '{database}' on server '{server}'.","@l":"Error","database":"tweetappapi","server":"tcp:tweetapp-server-atman.database.windows.net,1433","EventId":{"Id":20004,"Name":"Microsoft.EntityFrameworkCore.Database.Connection.ConnectionError"},"SourceContext":"Microsoft.EntityFrameworkCore.Database.Connection","ActionId":"4ba95a3c-2d2e-4fa7-b686-aa46ac338dae","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKNQ6I7E7I:00000001","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKKNQ6I7E7I"}
{"@t":"2022-09-12T13:15:30.1893951Z","@mt":"An exception occurred while iterating over the results of a query for context type '{contextType}'.{newline}{error}","@l":"Error","@x":"System.InvalidOperationException: An exception has been raised that is likely due to a transient failure. Consider enabling transient error resiliency by adding 'EnableRetryOnFailure' to the 'UseSqlServer' call.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): The instance of SQL Server you attempted to connect to requires encryption but this machine does not support it.\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CheckPoolBlockingPeriod(Exception e)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(AsyncEnumerator enumerator, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\nClientConnectionId:33b257a6-cf83-42ff-903c-1a56fb965a2f\r\nError Number:20,State:0,Class:20\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()","contextType":"TweetApp.Backend.Repository.ApplicationDbContext","newline":"\r\n","error":"System.InvalidOperationException: An exception has been raised that is likely due to a transient failure. Consider enabling transient error resiliency by adding 'EnableRetryOnFailure' to the 'UseSqlServer' call.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): The instance of SQL Server you attempted to connect to requires encryption but this machine does not support it.\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CheckPoolBlockingPeriod(Exception e)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(AsyncEnumerator enumerator, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\nClientConnectionId:33b257a6-cf83-42ff-903c-1a56fb965a2f\r\nError Number:20,State:0,Class:20\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()","EventId":{"Id":10100,"Name":"Microsoft.EntityFrameworkCore.Query.QueryIterationFailed"},"SourceContext":"Microsoft.EntityFrameworkCore.Query","ActionId":"4ba95a3c-2d2e-4fa7-b686-aa46ac338dae","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKNQ6I7E7I:00000001","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKKNQ6I7E7I"}
{"@t":"2022-09-12T13:15:30.3447455Z","@mt":"Invalid username or password","@l":"Error","SourceContext":"TweetApp.Backend.Controllers.UsersController","ActionId":"4ba95a3c-2d2e-4fa7-b686-aa46ac338dae","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKNQ6I7E7I:00000001","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKKNQ6I7E7I"}
{"@t":"2022-09-12T13:15:30.4390117Z","@mt":"HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms","@r":["1026.0041"],"RequestMethod":"POST","RequestPath":"/api/v1/tweets/login","StatusCode":200,"Elapsed":1026.0041,"SourceContext":"Serilog.AspNetCore.RequestLoggingMiddleware","ActionId":"4ba95a3c-2d2e-4fa7-b686-aa46ac338dae","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKNQ6I7E7I:00000001","ConnectionId":"0HMKKNQ6I7E7I"}
{"@t":"2022-09-12T13:18:45.4403104Z","@mt":"Starting web host"}
{"@t":"2022-09-12T13:18:47.7097587Z","@mt":"Storing keys in a directory '{path}' that may not be persisted outside of the container. Protected data will be unavailable when container is destroyed.","@l":"Warning","path":"C:\\Users\\ContainerUser\\AppData\\Local\\ASP.NET\\DataProtection-Keys","EventId":{"Id":60,"Name":"UsingEphemeralFileSystemLocationInContainer"},"SourceContext":"Microsoft.AspNetCore.DataProtection.Repositories.FileSystemXmlRepository"}
{"@t":"2022-09-12T13:18:48.2757227Z","@mt":"Now listening on: {address}","address":"https://[::]:443","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T13:18:48.2766683Z","@mt":"Now listening on: {address}","address":"http://[::]:80","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T13:18:48.2968037Z","@mt":"Application started. Press Ctrl+C to shut down.","SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T13:18:48.2973988Z","@mt":"Hosting environment: {envName}","envName":"Development","SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T13:18:48.2979304Z","@mt":"Content root path: {contentRoot}","contentRoot":"C:\\app\\","SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T13:18:59.7510279Z","@mt":"Entity Framework Core {version} initialized '{contextType}' using provider '{provider}:{providerVersion}' with options: {options}","version":"6.0.8","contextType":"ApplicationDbContext","provider":"Microsoft.EntityFrameworkCore.SqlServer","providerVersion":"6.0.8","options":"None","EventId":{"Id":10403,"Name":"Microsoft.EntityFrameworkCore.Infrastructure.ContextInitialized"},"SourceContext":"Microsoft.EntityFrameworkCore.Infrastructure","ActionId":"77060e9a-6288-4789-babf-796f4c7dacc3","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKO5A9QJHK:0000000F","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKKO5A9QJHK"}
{"@t":"2022-09-12T13:19:01.8835326Z","@mt":"An error occurred using the connection to database '{database}' on server '{server}'.","@l":"Error","database":"tweetappapi","server":"tcp:tweetapp-server-atman.database.windows.net,1433","EventId":{"Id":20004,"Name":"Microsoft.EntityFrameworkCore.Database.Connection.ConnectionError"},"SourceContext":"Microsoft.EntityFrameworkCore.Database.Connection","ActionId":"77060e9a-6288-4789-babf-796f4c7dacc3","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKO5A9QJHK:0000000F","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKKO5A9QJHK"}
{"@t":"2022-09-12T13:19:02.0141811Z","@mt":"An exception occurred while iterating over the results of a query for context type '{contextType}'.{newline}{error}","@l":"Error","@x":"System.InvalidOperationException: An exception has been raised that is likely due to a transient failure. Consider enabling transient error resiliency by adding 'EnableRetryOnFailure' to the 'UseSqlServer' call.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): The instance of SQL Server you attempted to connect to requires encryption but this machine does not support it.\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CheckPoolBlockingPeriod(Exception e)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(AsyncEnumerator enumerator, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\nClientConnectionId:ac3fa045-45b5-40a8-8451-0bbb1eb2cc9c\r\nError Number:20,State:0,Class:20\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()","contextType":"TweetApp.Backend.Repository.ApplicationDbContext","newline":"\r\n","error":"System.InvalidOperationException: An exception has been raised that is likely due to a transient failure. Consider enabling transient error resiliency by adding 'EnableRetryOnFailure' to the 'UseSqlServer' call.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): The instance of SQL Server you attempted to connect to requires encryption but this machine does not support it.\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CheckPoolBlockingPeriod(Exception e)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(AsyncEnumerator enumerator, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\nClientConnectionId:ac3fa045-45b5-40a8-8451-0bbb1eb2cc9c\r\nError Number:20,State:0,Class:20\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()","EventId":{"Id":10100,"Name":"Microsoft.EntityFrameworkCore.Query.QueryIterationFailed"},"SourceContext":"Microsoft.EntityFrameworkCore.Query","ActionId":"77060e9a-6288-4789-babf-796f4c7dacc3","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKO5A9QJHK:0000000F","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKKO5A9QJHK"}
{"@t":"2022-09-12T13:19:02.2588348Z","@mt":"Invalid username or password","@l":"Error","SourceContext":"TweetApp.Backend.Controllers.UsersController","ActionId":"77060e9a-6288-4789-babf-796f4c7dacc3","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKO5A9QJHK:0000000F","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKKO5A9QJHK"}
{"@t":"2022-09-12T13:19:04.5887809Z","@mt":"HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms","@r":["6803.5029"],"RequestMethod":"POST","RequestPath":"/api/v1/tweets/login","StatusCode":200,"Elapsed":6803.5029,"SourceContext":"Serilog.AspNetCore.RequestLoggingMiddleware","ActionId":"77060e9a-6288-4789-babf-796f4c7dacc3","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKO5A9QJHK:0000000F","ConnectionId":"0HMKKO5A9QJHK"}
{"@t":"2022-09-12T13:21:02.9073381Z","@mt":"Starting web host"}
{"@t":"2022-09-12T13:21:05.0812326Z","@mt":"Storing keys in a directory '{path}' that may not be persisted outside of the container. Protected data will be unavailable when container is destroyed.","@l":"Warning","path":"C:\\Users\\ContainerUser\\AppData\\Local\\ASP.NET\\DataProtection-Keys","EventId":{"Id":60,"Name":"UsingEphemeralFileSystemLocationInContainer"},"SourceContext":"Microsoft.AspNetCore.DataProtection.Repositories.FileSystemXmlRepository"}
{"@t":"2022-09-12T13:21:05.6550640Z","@mt":"Now listening on: {address}","address":"https://[::]:443","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T13:21:05.6558030Z","@mt":"Now listening on: {address}","address":"http://[::]:80","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T13:21:05.6600297Z","@mt":"Application started. Press Ctrl+C to shut down.","SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T13:21:05.6605586Z","@mt":"Hosting environment: {envName}","envName":"Development","SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T13:21:05.6608501Z","@mt":"Content root path: {contentRoot}","contentRoot":"C:\\app\\","SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T13:25:41.0861565Z","@mt":"Starting web host"}
{"@t":"2022-09-12T13:25:43.3995253Z","@mt":"Storing keys in a directory '{path}' that may not be persisted outside of the container. Protected data will be unavailable when container is destroyed.","@l":"Warning","path":"C:\\Users\\ContainerUser\\AppData\\Local\\ASP.NET\\DataProtection-Keys","EventId":{"Id":60,"Name":"UsingEphemeralFileSystemLocationInContainer"},"SourceContext":"Microsoft.AspNetCore.DataProtection.Repositories.FileSystemXmlRepository"}
{"@t":"2022-09-12T13:25:44.0371390Z","@mt":"Now listening on: {address}","address":"https://[::]:443","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T13:25:44.0380359Z","@mt":"Now listening on: {address}","address":"http://[::]:80","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T13:25:44.0663793Z","@mt":"Application started. Press Ctrl+C to shut down.","SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T13:25:44.0668180Z","@mt":"Hosting environment: {envName}","envName":"Development","SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T13:25:44.0670891Z","@mt":"Content root path: {contentRoot}","contentRoot":"C:\\app\\","SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2022-09-12T13:27:02.3487425Z","@mt":"Entity Framework Core {version} initialized '{contextType}' using provider '{provider}:{providerVersion}' with options: {options}","version":"6.0.8","contextType":"ApplicationDbContext","provider":"Microsoft.EntityFrameworkCore.SqlServer","providerVersion":"6.0.8","options":"None","EventId":{"Id":10403,"Name":"Microsoft.EntityFrameworkCore.Infrastructure.ContextInitialized"},"SourceContext":"Microsoft.EntityFrameworkCore.Infrastructure","ActionId":"d75c5b88-947d-4c0f-867a-37d29153604e","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKO975AGCS:00000001","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKKO975AGCS"}
{"@t":"2022-09-12T13:27:05.2384768Z","@mt":"An error occurred using the connection to database '{database}' on server '{server}'.","@l":"Error","database":"tweetappapi","server":"tcp:tweetapp-server-atman.database.windows.net,1433","EventId":{"Id":20004,"Name":"Microsoft.EntityFrameworkCore.Database.Connection.ConnectionError"},"SourceContext":"Microsoft.EntityFrameworkCore.Database.Connection","ActionId":"d75c5b88-947d-4c0f-867a-37d29153604e","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKO975AGCS:00000001","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKKO975AGCS"}
{"@t":"2022-09-12T13:27:05.4517305Z","@mt":"An exception occurred while iterating over the results of a query for context type '{contextType}'.{newline}{error}","@l":"Error","@x":"System.InvalidOperationException: An exception has been raised that is likely due to a transient failure. Consider enabling transient error resiliency by adding 'EnableRetryOnFailure' to the 'UseSqlServer' call.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): The instance of SQL Server you attempted to connect to requires encryption but this machine does not support it.\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CheckPoolBlockingPeriod(Exception e)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(AsyncEnumerator enumerator, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\nClientConnectionId:d65d0398-413b-410d-8f75-21b875035370\r\nError Number:20,State:0,Class:20\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()","contextType":"TweetApp.Backend.Repository.ApplicationDbContext","newline":"\r\n","error":"System.InvalidOperationException: An exception has been raised that is likely due to a transient failure. Consider enabling transient error resiliency by adding 'EnableRetryOnFailure' to the 'UseSqlServer' call.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): The instance of SQL Server you attempted to connect to requires encryption but this machine does not support it.\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CheckPoolBlockingPeriod(Exception e)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(AsyncEnumerator enumerator, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\nClientConnectionId:d65d0398-413b-410d-8f75-21b875035370\r\nError Number:20,State:0,Class:20\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()","EventId":{"Id":10100,"Name":"Microsoft.EntityFrameworkCore.Query.QueryIterationFailed"},"SourceContext":"Microsoft.EntityFrameworkCore.Query","ActionId":"d75c5b88-947d-4c0f-867a-37d29153604e","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKO975AGCS:00000001","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKKO975AGCS"}
{"@t":"2022-09-12T13:27:05.7219373Z","@mt":"Invalid username or password","@l":"Error","SourceContext":"TweetApp.Backend.Controllers.UsersController","ActionId":"d75c5b88-947d-4c0f-867a-37d29153604e","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKO975AGCS:00000001","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKKO975AGCS"}
{"@t":"2022-09-12T13:27:07.4815333Z","@mt":"HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms","@r":["7214.0630"],"RequestMethod":"POST","RequestPath":"/api/v1/tweets/login","StatusCode":200,"Elapsed":7214.063,"SourceContext":"Serilog.AspNetCore.RequestLoggingMiddleware","ActionId":"d75c5b88-947d-4c0f-867a-37d29153604e","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKO975AGCS:00000001","ConnectionId":"0HMKKO975AGCS"}
{"@t":"2022-09-12T18:16:25.2072516Z","@mt":"Entity Framework Core {version} initialized '{contextType}' using provider '{provider}:{providerVersion}' with options: {options}","version":"6.0.8","contextType":"ApplicationDbContext","provider":"Microsoft.EntityFrameworkCore.SqlServer","providerVersion":"6.0.8","options":"None","EventId":{"Id":10403,"Name":"Microsoft.EntityFrameworkCore.Infrastructure.ContextInitialized"},"SourceContext":"Microsoft.EntityFrameworkCore.Infrastructure","ActionId":"732378e9-cfc2-4813-865c-32d9cfbb6b55","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKSONUU77U:00000001","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKKSONUU77U"}
{"@t":"2022-09-12T18:16:34.1717652Z","@mt":"Invalid username or password","@l":"Error","SourceContext":"TweetApp.Backend.Controllers.UsersController","ActionId":"732378e9-cfc2-4813-865c-32d9cfbb6b55","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKSONUU77U:00000001","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKKSONUU77U"}
{"@t":"2022-09-12T18:16:35.9841784Z","@mt":"HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms","@r":["12750.7844"],"RequestMethod":"POST","RequestPath":"/api/v1/tweets/login","StatusCode":200,"Elapsed":12750.7844,"SourceContext":"Serilog.AspNetCore.RequestLoggingMiddleware","ActionId":"732378e9-cfc2-4813-865c-32d9cfbb6b55","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKKSONUU77U:00000001","ConnectionId":"0HMKKSONUU77U"}
